ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -219,11 +220,9 @@ Rectangle {
             sourceComponent: StatusMessageReply {
                 replyDetails: root.replyDetails
                 profileClickable: root.profileClickable
-                onReplyProfileClicked: root.replyProfileClicked(sender, mouse)
                 audioMessageInfoText: root.audioMessageInfoText
-                onLinkActivated: {
-                    root.linkActivated(link);
-                }
```

Comment:
URL links no longer supported?

Reply:
FWIW, discord supports that:
![image](https://user-images.githubusercontent.com/5377645/201316050-c73f6e32-634f-43a2-ad6b-414925b3a00e.png)

Reply:
There were discussions around that and consensus to remove styles and link activation altogether from reply-to-messages

Reply:
Ah alright, fair enough